pkg/localstore/reserve_test.go

```diff
@@ -646,8 +646,6 @@ func TestReserveSize(t *testing.T) {
 }
 
 func TestComputeReserveSize(t *testing.T) {
-	t.Parallel()
```

Comment:
not sure if accidental

Reply:
so this package doesnt have parallel tests. It shouldnt also as they override global variables. I think this was added by mistake but there should not be any t.Parallel tests in localstore for now.